CONTRIBUTING.md

```diff
@@ -0,0 +1,115 @@
+# [<img src="https://avatars1.githubusercontent.com/u/3771091?v=3&s=30">](https://github.com/gonum) Contribution Guidelines
+#### Introduction
+
+The Gonum project provides general purpose numerical computing and data science libraries for the Go programming language, and we would like you to join us in improving Gonum's quality and scope.
+This document is for anyone who is contributing or interested in contributing.
+
+#### Table of Contents
+
+[Getting Started](#getting-started)
+  * [Working Together](#working-together)
+  * [Project Goals](#project-goals)
+  * [The Gonum Repos](#the-gonum-repos)
+
+[How to Contribute](#how-to-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+  * [Style](#style)
+
+## Getting Started
+
+Gonum discussion takes place on the [gonum-dev google group](https://groups.google.com/forum/#!forum/gonum-dev).
```

Comment:
Code is at github.com/gonum/\<repo\>.
Documentation is at godoc.org/github.com/gonum/\<repo\>.